docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1876,18 +1878,14 @@ for who files taxes:
 
 
 **Data Errors/Noise** 
-In the future, we will add a noise function designed to replicate 
-missed or incorrect data. This includes incorrect name spelling, 
-addresses, age, and person swaps. 
 
-Some additional tax specific errors include: SSN randomly being filled 
-for those who are missing one, mailing address being different than the 
-home address, and issues with employer names, addresses, or IDs. 
-
-Furthermore, there are often complications with people being claimed as 
-dependents in different households, or claiming people from group quarters.
+.. todo::
+    Engineers: these things are not yet defined and might be edited later - they should be parameterizable 
 
-These are not currently modeled. 
+    - Omission rates, oversampling might be needed to avoid rerunning 
+    - Creation of a noise function for misrecording data (names, addresses, birthdays) 
+    - Further refine "sharing" of SSN's 
+    - Include person swaps or duplicates - especially relevant for taxes if dependents should be double claimed sometimes 
```

Comment:
I think we need to specify what we might need for these last two bullets especially.